wp-content/themes/core/blocks/core/postterms/style.pcss

```diff
@@ -2,10 +2,50 @@
  * Styles specific to this block
  */
 
-.wp-block-post-terms {
+.wp-block-post-terms,
+:is(.editor-styles-wrapper) .wp-block-post-terms {
 
 	> a {
 
 		@mixin t-category;
+		color: var(--color-black);
+
+		&:hover,
+		&:focus {
+			text-decoration: underline;
+		}
+	}
+
+	/**
+	 * Adding the .is-tags class to the post terms block will give pill shaped
+	 * clickable tags: http://p.tri.be/i/K0SeDF
+	 * Post terms block adds forced separator with an empty space if nothing is
+	 * defined in the "Separator" field in the block editor, so the gap of 2px
+	 * is set to define a small space between each tag and the separator (for
+	 * a total of ~8px between each tag)
+	 */
+	&.is-tags {
```

Comment:
🟠  looks like `taxonomy-post_tag` is added to the block. Was there a reason to use a custom class vs targeting what comes from WP? I'd prefer if we targeted that vs devs/editors knowing that class controls something